src/containers/FriendListPage.js

```diff
@@ -29,6 +30,13 @@ class FriendListPage extends PureComponent {
     };
   }
 
+  componentDidMount() {
+    let { myId, friendListPage, actions: {doRootPage} } = this.props
+
+    let rootPageId = friendListPage.getIn([myId, 'parentId'])
+    doRootPage.markFriendListSeen(rootPageId)
```

Comment:
I think it'd be better to move `markFriendListSeen` up to RootPage.js.

Or alternatively, consider using `markSeen()` in RootPage.js.

Reply:
agreed.